ReactSkia/sdk/CurlNetworking.cpp

```diff
@@ -116,13 +116,11 @@ void CurlNetworking::processNetworkRequest(CURLM *curlMultiHandle) {
             (curlRequest->curlResponse->responseTimeout = true)
             :(curlRequest->curlResponse->responseTimeout = false);
         curl_multi_remove_handle(curlMultiHandle, curlHandle);
-        char *url = NULL;
-        curl_easy_getinfo(curlHandle, CURLINFO_EFFECTIVE_URL, &url);
```

Comment:
Effective url is still required to be sent as part of didReceiveResponse event, because XHR responseUrl expects the final URL after redirects
https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/responseURL

Reply:
yes, Effective URL is required to be sent as part of didReceiveResponse event, but not in setCache. In cache we will always cache Requested URL. As part of didReceiveResponse event, we are still sending Effective URL